app/src/main/res/values/strings.xml

```diff
@@ -361,36 +361,36 @@
     <string name="error_no_activity_to_view">Could not find an app to open the content with</string>
     <string name="manage_clients">Manage Clients</string>
     <string name="apps">Apps</string>
-    <string name="generate">Generate</string>
-    <string name="scan">Scan</string>
+    <string name="generate">Generate…</string>
```

Comment:
This is a button.

![Screenshot_20210829-080138_TShot_DEBUG](https://user-images.githubusercontent.com/10258162/131239056-141bbf14-d50e-4dfe-b798-61d584253d3c.png)

Reply:
```suggestion
    <string name="generate">Generate</string>
```